esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -4111,6 +4111,39 @@ void CWsWuFileHelper::createWULogFile(IConstWorkUnit *cwu, WsWuInfo &winfo, cons
 }
 
 #ifdef _CONTAINERIZED
+template <class T>
+void readWUComponentLogOptionsReq(T* logReq, WUComponentLogOptions& options)
```

Comment:
Trivial: You could still have used a reference for the 1st parameter - slightly better.

Reply:
Will change to a reference.

Reply:
ok, but I just to make sure you knew I didn't think it was required - since I approved it as it was.

Reply:
@ghalliday I do like your suggestion. I have made the changes in https://github.com/hpcc-systems/HPCC-Platform/pull/16285/commits/9dd9a2f079bac3e2ddce8e70adc550d04c890c4b and tested. Please take a look before I rebase the changes.